_posts/2019-08-02-Rates-Rebates-Recommendations-Report.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Recommendations Report
+date: 2019-08-02
+authors:
+  - glen-thurston
+tags:
+  - blog post
+  - open source
+  - rates rebates
```

Comment:
```suggestion
  - rates rebate
```

Reply:
i think it's supposed to be plural.

Reply:
![Screenshot from 2019-10-25 10-30-54](https://user-images.githubusercontent.com/13788498/67528001-d109d580-f714-11e9-8c42-243e0497afda.png)
I'm getting conflicting search results.
But went with singular b'cos people only get **A** rebate (out of their rates)?